integration-tests/src/test/java/oracle/weblogic/kubernetes/ItFmwBigCMMiiDomain.java

```diff
@@ -0,0 +1,425 @@
+        "Failed to copy file " + srcModelFile + " to file " + targetModelFile);
+
+    assertDoesNotThrow(() -> replaceStringInFile(targetModelFile.toString(),
+        "BIGDATAREPLACE",
+        propVal.toString()), "Can't replace the string BIGDATAREPLACE in " + targetModelFile);
+    assertDoesNotThrow(() -> replaceStringInFile(targetModelFile.toString(),
+        "@@PROP:K8S_NODEPORT_HOST@@:@@PROP:DBPORT@@",
+        String.format("%s:%s", K8S_NODEPORT_HOST, Integer.toString(dbNodePort))),
+        "Can't replace the string @@PROP:K8S_NODEPORT_HOST@@:@@PROP:DBPORT@@ in " + targetModelFile);
+
+    final List<String> modelList = Collections.singletonList(targetModelFile.toString());
+    String jrfMii1Image = createMiiImageAndVerify(
+        "jrf-mii-image",
+        modelList,
+        Collections.singletonList(MII_BASIC_APP_NAME),
+        FMWINFRA_IMAGE_NAME,
+        FMWINFRA_IMAGE_TAG,
+        "JRF",
+        false);
+
+    // push the image to a registry to make it accessible in multi-node cluster
+    dockerLoginAndPushImageToRegistry(jrfMii1Image);
+
+    // create the domain object
+    Domain domain = createDomainResource(domainUid,
+        jrfDomainNamespace,
+        adminSecretName,
+        OCIR_SECRET_NAME,
+        encryptionSecretName,
+        rcuaccessSecretName,
+        opsswalletpassSecretName,
+        replicaCount,
+        jrfMii1Image);
+
+    createDomainAndVerify(domain, jrfDomainNamespace);
+    verifyDomainReady(jrfDomainNamespace, domainUid);
+    // check if multiple configmaps are created
+    try {
+      if (!Kubernetes.listConfigMaps(jrfDomainNamespace).getItems().isEmpty()) {
+        logger.info("Getting Config Maps List");
+        int cmTotalSize = 0;
+        List<V1ConfigMap> items = Kubernetes.listConfigMaps(jrfDomainNamespace).getItems();
+        List<V1ConfigMap> itemsCM = new ArrayList<>();
+        for (var item : items) {
+          if (item.getMetadata().getName().contains("introspect")) {
+            logger.info("Found ConfigMap " + item.getMetadata().getName());
+            logger.info("Found ConfigMap size " + item.toString().getBytes("UTF-8").length + " bytes");
+            cmTotalSize = cmTotalSize + item.toString().getBytes("UTF-8").length;
+            itemsCM.add(item);
+          }
+        }
+        assertTrue((cmTotalSize > 1000000) && (itemsCM.size() > 1),
+            "Produced introspector domain cm is bigger than 1M and was not splitted");
+      }
+    } catch (Exception ex) {
+      throw new RuntimeException("Failed to process config maps" + ex.getMessage());
+    }
+  }
+
+  /**
+   * Save the OPSS key wallet from a running JRF domain's introspector configmap to a file.
+   * @param namespace namespace where JRF domain exists
+   * @param domainUid unique domain Uid
+   * @param walletfileSecretName name of wallet file secret
+   */
+  private void saveAndRestoreOpssWalletfileSecret(String namespace, String domainUid,
```

Comment:
This method is already there in ItFmwMiiDomain.java, so this is another candidate to move into a Utility class say FmwUtils.java

Reply:
done